File: main/CmdType.go
package main

type CmdType int

const (
	createDb    CmdType = iota // value --> 0
	createTable                // value --> 1
)

func (this CmdType) String() string {
	switch this {
	case createDb:
		return "Running"
	case createTable:
		return "Stopped"
	default:
		return "Unknow"
	}
}

File: java/src/main/java/com/earyant/database/explain/CmdType.java
package com.earyant.database.explain;

public enum CmdType {
    createTable(0),
    createDb(1),
    insertData(2),
    ;

    Integer value;
    CmdType(int i) {
        this.value = i;
    }

    public Integer getValue() {
        return value;
    }

    public void setValue(Integer value) {
        this.value = value;
    }
}
